luxonis_train/core/utils/infer_utils.py

```diff
@@ -28,3 +33,70 @@ def render_visualizations(
     if save_dir is None:
         if cv2.waitKey(0) == ord("q"):
             exit()
+
+
+def process_single_image(
+    model, img_path: Path, view: str, save_dir: str | Path | None
+) -> None:
+    """Handles the inference on a single image."""
+    img = cv2.cvtColor(cv2.imread(str(img_path)), cv2.COLOR_BGR2RGB)
+    img, _ = model.val_augmentations([(img, {})])
```

Comment:
Since we already have `view` passed here can we also set augmentations depending on this value (if train use train augs otherwise val)? Could be nice for cases where you maybe want to infer on images with some specific augmentations.